<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="qsg-PC" name="requirementGroup.SettingUpResources.NewImportSubResourcesFromSpreadsheet" tests="1" failures="0" timestamp="9 Sep 2015 07:21:33 GMT" time="434.366" errors="1">
  <testcase name="testHappyDay107892" time="434.366" classname="requirementGroup.SettingUpResources.NewImportSubResourcesFromSpreadsheet">
    <error type="java.lang.AssertionError" message="expected [ImportResource.xls] but found [D:\eclipse\com.qsgsoft.EMResourceWebDriver\src\test\resources\UploadFiles\ImportResource.xls]">
      <![CDATA[java.lang.AssertionError: expected [ImportResource.xls] but found [D:\eclipse\com.qsgsoft.EMResourceWebDriver\src\test\resources\UploadFiles\ImportResource.xls]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:494)
	at org.testng.Assert.assertEquals(Assert.java:123)
	at org.testng.Assert.assertEquals(Assert.java:176)
	at org.testng.Assert.assertEquals(Assert.java:186)
	at lib.page.Upload.verUploadedFilename(Upload.java:235)
	at requirementGroup.SettingUpResources.NewImportSubResourcesFromSpreadsheet.testHappyDay107892(NewImportSubResourcesFromSpreadsheet.java:195)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1224)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1149)
	at org.testng.TestNG.run(TestNG.java:1057)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:111)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:204)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:175)
]]>
    </error>
  </testcase> <!-- testHappyDay107892 -->
</testsuite> <!-- requirementGroup.SettingUpResources.NewImportSubResourcesFromSpreadsheet -->
